### PR TITLE
LPS-173193 Avoid NullPointerException

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-173193

### CODE REVIEW

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java [hunk 1]
```diff
@@ -140,7 +140,15 @@ public void sendNotification(NotificationContext notificationContext)
...
+		Group userGroup = user.getGroup();
+
+		long groupId = 0;
+
+		if (userGroup != null) {
+			groupId = userGroup.getGroupId();
+		}
+
+		siteDefaultLocale = portal.getSiteDefaultLocale(groupId);
 		userLocale = user.getLocale();
```
Comment (carolmariaabb) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java:152]: @marcelabc I would just add a blank line after `siteDefaultLocale = portal.getSiteDefaultLocale(groupId);`

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**